examples/cfd/08_channel_flow.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<p>#Silence non-essential outputs from the solver.</p><p> </p><p>We need to add space after the comment.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Fixed.